implementation/objectoriented.tex

```diff
@@ -179,11 +182,17 @@ \subsection{Memory Layout of Objects}
 By comparing \autoref{fig:back:memorymapped} and \autoref{fig:back:memlayout} we see that the memory layout of a struct defined in {\rust} and {\C} has same exact layout as the memory mapped control register of the \gls{adc}.
 This suggests that, if a pointer to the \gls{mmio} device is considered as a reference to an \gls{adc} object, the object oriented pattern can be used to directly interface with the \gls{mmio}.
 
+The layout of the Java object in \autoref{fig:back:memlayout:java} could imply that the same analysis can be applied by adding an offset, equal to the size of the object tag, to the reference.
+This is not the case as this would map the object tag to the base address of the \gls{adc} minus 4 bytes.
+This location is, in the case of the {\gecko}, an unmapped memory section used to as padding the previous \gls{mmio}.
```

Comment:
muligens endre til: 
"This location is, in the case of the {\gecko}, an unmapped memory section that is used for padding between the ADC and the previous MMIO."